### PR TITLE
Can subroutine haz abi (in graviton tests)

### PR DESCRIPTION
**Pull request by tzaffi**
When reviewing #473 I relaized that the `PyTealDryRunExecutor` hadn't been updated to reflect the fact that `@Subroutines` may happily accept ABI-types. Consequently, there were all sorts of contortions necessary to get graviton tests to run on the `NamedTuple` example. 

This change should simplify usage of graviton tests in the case of regular `@Subroutines` with ABI-types.

### CODE REVIEW

File: tests/integration/graviton_test.py [hunk 1]
```diff
@@ -961,22 +960,28 @@ def named_tuple_field_access(
...
+    #     ("bool[4]", [False] * 4),
+    #     ("uint64", 100),
+    # ]
+
+    # args: list[bytes] = [
+    #     ABIType.from_string(abi_t).encode(var) for abi_t, var in TYPES_N_VALS
+    # ]
+
+    # lsig_result = lsig_pytealer.dryrun(args)
+    # assert lsig_result.passed()
```
Comment (ahangsu) [inline tests/integration/graviton_test.py:984]: if graviton can do all the stuffs from python var to ABI encodings, that is great! I don't see any point we should keep my previous snippet of implementation here tho...

Comment (tzaffi) [reply tests/integration/graviton_test.py:984]: ok, I'll update without

Comment (tzaffi) [reply tests/integration/graviton_test.py]: updated

Comment (ahangsu) [reply tests/integration/graviton_test.py]: 👍

**Review by ahangsu — APPROVED** [review]

**Pull request merged**